From ae1e9c7e5be6cbe670fe41c53b49743673eec549 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 14 Dec 2016 17:07:01 +0900
Subject: [PATCH 1425/2066] arm64: dts: r8a7795: Rename of the reference clock
 name of DU

This patch changes to the reference clock names of DU changed by
commit 6410550c1ae7 ("arm64: dts: r8a7795: Fix clock node name in DU").

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
index 584a2a7..2a5fc3d 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
@@ -235,19 +235,19 @@
 	};
 };
 
-&programmable_clk0 {
+&du_dotclkin0 {
 	clock-frequency = <148500000>;
 };
 
-&x21_clk {
+&du_dotclkin1 {
 	clock-frequency = <33000000>;
 };
 
-&x22_clk {
+&du_dotclkin2 {
 	clock-frequency = <33000000>;
 };
 
-&programmable_clk1 {
+&du_dotclkin3 {
 	clock-frequency = <108000000>;
 };
 
-- 
1.7.5.4

